src/runtime-rs/crates/runtimes/common/src/types/mod.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+mod trans_from_shim;
+mod trans_into_shim;
+
+use std::fmt;
+
+use anyhow::{Context, Result};
+use kata_sys_util::validate;
+use kata_types::mount::Mount;
+
+/// Request: request from shim
+/// Request and Response messages need to be paired
+#[derive(Debug, Clone)]
+pub enum Request {
+    CreateContainer(ContainerConfig),
+    CloseIo(ProcessID),
```

Comment:
Nit: Since `IO` is an abbreviation (and `Io` is a satellite 😄), I suggest to spell `CloseIO`

Reply:
Thanks for your review, already update it.